asn1/GenericPacket.asn

```diff
@@ -35,13 +67,18 @@ PacketSet PACKET ::= {
     {6 QuoteByDestinationAmountRequest} |
     {7 QuoteByDestinationAmountResponse} |
     {8 InterledgerProtocolError}
+
+    {253 CustomRequest} |
+    {254 CustomResponse} |
```

Comment:
Arguably if this message is going to be riding on a request or response, you just need a `CustomMessage` type like in https://github.com/interledger/rfcs/pull/260/files#diff-42c9b615e907424c7cae3feb333d8b6fR75